File: ProjectEuler/007/007.cpp
#include <iostream>
#include <cmath>
using namespace std;

bool is_prime(int n){
	for(int i = 2; i < n; i++){
		if(n % i == 0) return false;
	}
	return true;
}

int main(){
	int p;
	int n = 0;
	int i = 2;
	while(n<10001){
		if(is_prime(i)){
			p = i;
			n++;
		}
		i++;
	}
	cout << p;

	return 0;
}

File: ProjectEuler/007/007.py
import math

def is_prime(n):
	for i in range(2, n):
		if(n % i == 0): return False
	return True

p = 0
n = 0
i = 2
while n < 10001:
	if(is_prime(i)):
		p = i
		n+=1
	i+=1
print(p)
